src/www/ui_tests/api/Controllers/GroupControllerTest.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/***************************************************************
+ * Copyright (C) 2020 Siemens AG
```

Comment:
Please remove Siemens' copyright and add yours 😃

Reply:
Done, anything else?